rules-reviewed/eap7/eap71/hibernate51-53.windup.xml

```diff
@@ -224,17 +224,24 @@
                 </hint>
             </perform>
         </rule>
-        <!--https://issues.jboss.org/browse/WINDUPRULE-381
+        <!--https://issues.jboss.org/browse/WINDUPRULE-381 -->
         <rule id="hibernate51-53-00500">
             <when>
+                <javaclass references="org.hibernate.{className}" >
```

Comment:
I think it's worth adding the `<location>` tag to just have the rule to match on imports and not every where in the classes.
Consider this a kind of trick to avoid having too many incidents per class and let the user focus on one incident (identified in the import) per each class.
```suggestion
                <javaclass references="org.hibernate.{className}" >
                    <location>IMPORT</location>
```